### PR TITLE
[4.0] Fix toolbar collapse in mobile view

### PR DESCRIPTION
**Pull request by Quy**
Fixes #27958.

### Summary of Changes
Use the correct class to fix toolbar collapse in mobile view.

### Testing Instructions
Apply PR.
Run npm i or install the package installer: https://ci.joomla.org/artifacts/joomla/joomla-cms/4.0-dev/30131/downloads/35475/
Resize browser's viewport.
Go to Global Configuration.
Go to any list views.

### Actual result BEFORE applying this Pull Request
The toolbar toggle button requires 2 clicks to close. The first click attempts to open the already open toolbar.

### Expected result AFTER applying this Pull Request
Toolbar collapsed by default.

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 91ff85b36279feca465dac1daa753b173d63e748<br /><br />PR works as decribed.

But there is another issue with the toolbar collapse not related to this PR, i.e. it is there with and without this PR.

When resizing the browser window, the toolbar collapsing and the gears button display doesn't happen at the same time. Especially when enlarging the window, first the gears button disappears, but for making the tool bar appear it needs to further enlarge the window, i.e. there is a small diplay width range in with neither the gears button to expand or collapse the toolbar nor the expanded toolbar are shown.

@Quy I guess that's a css problem. Do you think you can fix that, too? And do you know if there is an issue for that?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30131">issues.joomla.org/tracker/joomla-cms/30131</a>.</sub>

**Comment by Quy** [comment]
I am not at computer to test, but #30132 should have fixed the issue with the collapsed toolbar.

**Comment by richard67** [comment]
Ahh, so both PRs together should do the job.

**Comment by richard67** [comment]
@Quy No, the issue persists when I apply the changes of both PRs (this one here and #30132 ).

For display widths of 768 px or larger, the tool bar is shown, all fine.
j4-toolbar-w768


For display widths between 576 and 767 px, the toolbar is not shown, but also not the button for it.

Width 767 px:
j4-toolbar-w767

Width 576 px:
j4-toolbar-w576


For display widths of 757 px or smaller, the button is shown:
j4-toolbar-w575

File: build/media_source/templates/atum/js/template.es6.js [hunk 1]
```diff
@@ -211,11 +211,11 @@
     }
 
     if (smallLandscape.matches) {
```
Comment (Quy) [inline build/media_source/templates/atum/js/template.es6.js]: Can you see if changing to `if (small.matches) {` fixes the issue?

Comment (richard67) [reply build/media_source/templates/atum/js/template.es6.js]: @Quy This fixes the problem which I've mentioned with the toolbar and its gears button, but it creates the same problem with the left side menu and its burger button, i.e. when resizing from small to a bit larger, the burger button disappears, but the (collapsed) menu doesn't appear. I think this was ok before the changes. It seems to me the 2 buttons are handled with the same css class but the toolbar and the menu have different sizes when they get collapsed or hidden.

Comment (Quy) [reply build/media_source/templates/atum/js/template.es6.js]: Added the suggested change, but able to reproduce your finding. Can you try again and advise which pixels causing the issue? Thanks.

**Comment by particthistle** [comment]
Went to test this but the link to the PR files: https://ci.joomla.org/artifacts/joomla/joomla-cms/4.0-dev/30131/downloads/34008/ Gives File not found on all the downloads :(

Also when I went to apply the PR there was nothing able to be applied.


_Modified per BFH discussion to indicate the failed test was not an actual test._

**Comment by richard67** [comment]
> Gives File not found on all the downloads :(

> Also when I went to apply the PR there was nothing able to be applied.

@particthistle Both is not fault of this PR. The first one is caused by an erroneout cleanup script having cleaned up the wrong stuff, the 2nd thing is a patchtester thing.

So please change your test result back to "not tested" (can be done in the issue tracker by using the "Test this" button again).

It can never be the fault of a PR when something is wrong with inbfrastructure or test tools.

**Comment by particthistle** [comment]
I have not tested this item.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30131">issues.joomla.org/tracker/joomla-cms/30131</a>.</sub>

**Comment by richard67** [comment]
@Quy I've just tested again and my [comment above](https://github.com/joomla/joomla-cms/pull/30131#discussion_r457222345) is still valid:

>  when resizing from small to a bit larger, the burger button disappears, but the (collapsed) menu doesn't appear.

**Comment by richard67** [comment]
@Quy P.S.: But it seems to happen only after the first reducing and enlarging again the width. Ehen on the same page and again reducing and then enlarging width, it works.

**Comment by richard67** [comment]
@Quy P.P.S.: If then going to another page and doing the same, it also happens only for the first time when reducing and then enlarging again the width. After that and a page reload, it works well on that page, until you have been on another page again.

**Comment by richard67** [comment]
@Quy It seems the issue I've found is not related to this PR. Will continue to investigate and if necessary make new issue or PR. Could you fix the javascript cs error reported here [https://ci.joomla.org/joomla/joomla-cms/35436/1/23](https://ci.joomla.org/joomla/joomla-cms/35436/1/23) ? After that I can give this PR a good test.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 22f88f08201025ff0e733a5276de261bb425771f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30131">issues.joomla.org/tracker/joomla-cms/30131</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 22f88f08201025ff0e733a5276de261bb425771f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30131">issues.joomla.org/tracker/joomla-cms/30131</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30131">issues.joomla.org/tracker/joomla-cms/30131</a>.</sub>

**Comment by infograf768** [comment]
Tks

**Pull request merged**